README.md

```diff
@@ -1,586 +1,15 @@
-  - [Checking the DPS Job Status](#checking-the-dps-job-status)
-  - [Getting the DPS Job Results](#getting-the-dps-job-results)
-- [Getting the GeoJSON URL for a geoBoundary](#getting-the-geojson-url-for-a-geoboundary)
-- [Contributing](#contributing)
-  - [Development Setup](#development-setup)
-  - [Creating an Algorithm Release](#creating-an-algorithm-release)
-  - [Registering an Algorithm Release](#registering-an-algorithm-release)
-- [Citations](#citations)
+## History
+The GEDI Subsetter tool was developed in response to the release of GEDI data and the subsequent interest from the biomass group. The group expressed interest in studying an entire region, which required obtaining all granules that cover that area. Due to GEDI products being an orbit of the International Space Station (ISS), the challenge was to efficiently subset all granules passing through the area of interest. So while the existing methods would work for a small number of granules, the users were seeking a more streamlined approach that would allow them to efficiently gather their data points without the need to process each file separately.
 
-## Algorithm Outline
+## About GEDI
+The Global Ecosystem Dynamics Investigation (GEDI) is a lidar instrument on the International Space Station that generates high-resolution laser ranging observations of the 3D features of the Earth. GEDI’s precise measurements of forest canopy height, canopy vertical structure, and surface elevation greatly advance our ability to characterize important carbon and water cycling processes, biodiversity, and habitat.
 
-At a high level, the GEDI subsetting algorithm does the following:
+## The Subsetter
+The Subsetter utilizes the Multi-Mission Algorithm and Analysis Platform’s (MAAP) async job queuing system that runs on AWS. The tool automatically handles authentication (AWS S3 direct access) and auto-refreshes tokens. Users are able to pass a number of inputs to the Subsetter’s job to refine the results. By querying NASA Common Metadata Repository (CMR) to gather all the granules that pass through a specified region, the Subsetter is able to parallelize the process to download the granules and then append the subsetted results to a single geopackage.
 
-- Queries the MAAP CMR for granules from a specified GEDI collection that
-  intersect a specified Area of Interest (AOI) (as a GeoJSON file).  This is
-  limited to GEDI collections granule data files are in HDF5 format, which are
-  L1B, L2A, L2B, and L4A.
-- For each granule within the specified AOI, downloads its HDF5 (`.h5`) file.
-- Subsets each data file by selecting specified datasets within the file and
-  limiting data to values that match a specified query condition.
-- Combines all subset files into a single output file named `gedi_subset.gpkg`,
-  in GeoPackage format, readable with `geopandas` as a `GeoDataFrame`.
+To read more specifics about the Subsetter: [Specification README](src/gedi_subset/Specification_README.md).
```

Comment:
Good idea to split this out. Let's name the file `MAAP_USAGE.md`. See also my comment on the file itself for further splitting.